post-install.xq

```diff
@@ -60,7 +60,7 @@ xmldb:reindex($config:logs-col),
 if (doc-available($config:raw-packages-doc) and doc-available($config:package-groups-doc)) then
     ()
 else
-    scanrepo:rebuild-all-package-metadata(),
+    scanrepo:rebuild-all-package-metadata() ! sm:chown(., "repo"),
```

Comment:
Instead of hard-coding the username, might it be preferable to `config:repo-permissions()?user`?

Reply:
Yup, that's way better